examples/double_pendulum/double_pendulum.py

```diff
@@ -1,5 +1,4 @@
 from sympy import symbols
-from sympy.physics.mechanics import *
```

Comment:
It should be deleting these `import *`. Either fix them so it isn't `import *` or leave them be.